libpod/container_internal.go

```diff
@@ -1708,6 +1708,14 @@ func (c *Container) mountStorage() (_ string, deferredErr error) {
 	tz := c.Timezone()
 	if tz != "" {
 		timezonePath := filepath.Join("/usr/share/zoneinfo", tz)
+
```

Comment:
Should all of this come from the similar code in containers/common?

Reply:
I would say yes, especially since we have a clone of that code in https://github.com/cri-o/cri-o/pull/7589. I stated that in https://github.com/cri-o/cri-o/issues/7550#issuecomment-1840409824. For CRI-O we would like to still maintain our own config field but reuse the helper functions per https://github.com/cri-o/cri-o/issues/7550#issuecomment-1841182649.